programs/drift/src/state/order_params.rs

```diff
@@ -630,6 +630,81 @@ impl OrderParams {
     }
 }
 
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default, Eq, PartialEq, Debug)]
+pub struct SwiftOrderParamsMessage {
```

Comment:
if we just put the expected_order_id on this struct, can we reuse the OrderParams struct?

Reply:
i did this so that way it could generalize a bit easier in the future if a bundle were to accept multiple orders at once. but given where we are now, i can do this yes and then we can look to change it in the future if need be